src/libsync/syncfileitem.cpp

```diff
@@ -81,8 +81,10 @@ SyncJournalFileRecord SyncFileItem::toSyncJournalFileRecordWithInode(const QStri
 
     // Some types should never be written to the database when propagation completes
     rec._type = _type;
-    if (rec._type == ItemTypeVirtualFileDownload)
+    if (rec._type == ItemTypeVirtualFileDownload) {
         rec._type = ItemTypeFile;
+        qCWarning(lcFileItem) << "Changing item type from virtual to normal file";
```

Comment:
maybe  also add the content of the comment a few lines before to the debug output
would probably speed up any future analyze of logs

Reply:
replied on the new version of the code